src/main/java/ch/njol/skript/classes/data/DefaultFunctions.java

```diff
@@ -500,6 +502,22 @@ public ColorRGB[] executeSimple(Object[][] params) {
 		}).description("Returns a RGB color from the given red, green and blue parameters.")
 			.examples("dye player's leggings rgb(120, 30, 45)")
 			.since("2.5");
+
+		Functions.registerFunction(new SimpleJavaFunction<OfflinePlayer>("player", new Parameter[] {
+			new Parameter<>("nameOrUUID", DefaultClasses.STRING, true, null)
```

Comment:
Would it be possible to handle if a player entered a UUID object by mistake?

Reply:
I don't think we have a UUID object in Skript, it is all handled as string though I will double check.